questionnaire/dataset/dataset-routes.test.js

```diff
@@ -10,7 +10,7 @@ const tokens = {
     'read:questionnaires':
         'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJkYXRhLWNhcHR1cmUtc2VydmljZSIsImlzcyI6IiQuYXVkIiwianRpIjoiN2YwYjAxOTktMjcxOS00ZGUxLWE1MjgtY2RmMjZkNmM3YTU5Iiwic3ViIjoiJC5hdWQiLCJzY29wZSI6InJlYWQ6cXVlc3Rpb25uYWlyZXMiLCJpYXQiOjE1NzAxODE2NDB9.uoGJM6snG6pd51UMbf7k86_1ACIrGdEJBRjlEv0twIQ'
 };
-const filepath = 'openapi/src/json-schemas/api/_questionnaires_{questionnaireId}_dataset/get/res';
+const filepath = 'openapi/src/json-schemas/api/v2_questionnaires_{questionnaireId}_dataset/get/res';
```

Comment:
Is this correct? I can see that this v2_questionnaires directory has been deleted